python-main.js

```diff
@@ -286,6 +286,7 @@ function translations() {
         $('.roundbutton').each(function(object, value) {
             var button_id = $(value).attr('id');
             $(value).attr('title', buttons[button_id]['title']);
+            $(value).attr('aria-label', buttons[button_id]['aria-label']);
```

Comment:
Shouldn't this be `buttons[button_id]['title']`?
I don't think there is strings in the language js files for `aria-label`, and if the text is going to be the same as the `title` we can reuse them.